node/pkg/node/node_test.go

```diff
@@ -511,8 +512,11 @@ func waitForStatusServer(ctx context.Context, logger *zap.Logger, statusAddr str
 }
 
 func TestMain(m *testing.M) {
+	logger, _ = zap.NewDevelopment()
```

Comment:
This seems to be used in parts of test code but not the code being tested https://github.com/wormhole-foundation/wormhole/blob/c0e097c12485f032579ee14e3093be90c4b76eb0/node/pkg/node/node_test.go#L226. Is this on purpose?

Reply:
I was not trying to refactor all of the code in this PR, but will definitely have some followups once this is merged. I'm trying to do the bare minimum to safely get us on a newer version of go. @bruce-riley suggested just setting up a global logger for most of the tests for this and that did help work around some of the inadequacies in the supervisor.